ohapi/cassettes/test_update_data_valid_master_access_token.yaml

```diff
@@ -0,0 +1,52 @@
c-28e8-11e8-b6b5-feb908e75a1b/foo?Signature=jty3AZjaRTq6T5XkxZoyH1nyqaE%3D&Expires=1521442387&AWSAccessKeyId=AKIAIKNTFUJJTNS6N7HA","metadata":{"tags":["test_tag"],"description":"test_file"},"source":"direct-sharing-91"},{"id":34940,"basename":"foo","created":"2018-03-15T17:30:28.922263Z","download_url":"https://open-humans-production.s3.amazonaws.com/member-files/direct-sharing-91/8df45266-2876-11e8-b199-b6e6950a685d/foo?Signature=uUzUyTltZZdqT9UFbTqz4kRXbic%3D&Expires=1521442387&AWSAccessKeyId=AKIAIKNTFUJJTNS6N7HA","metadata":{"tags":["test_tag"],"description":"test_file"},"source":"direct-sharing-91"},{"id":34930,"basename":"foo","created":"2018-03-15T15:35:18.847415Z","download_url":"https://open-humans-production.s3.amazonaws.com/member-files/direct-sharing-91/7744c326-2866-11e8-b199-b6e6950a685d/foo?Signature=tm8pw4A%2F%2Ftp5ZGVg0PlASHI7b0A%3D&Expires=1521442387&AWSAccessKeyId=AKIAIKNTFUJJTNS6N7HA","metadata":{"tags":["test_tag"],"description":"test_file"},"source":"direct-sharing-91"},{"id":34929,"basename":"foo","created":"2018-03-15T15:34:17.838538Z","download_url":"https://open-humans-production.s3.amazonaws.com/member-files/direct-sharing-91/52d0f19a-2866-11e8-b199-b6e6950a685d/foo?Signature=xxXbj8TpBfqhM0oB0AaxF4B7B1k%3D&Expires=1521442387&AWSAccessKeyId=AKIAIKNTFUJJTNS6N7HA","metadata":{"tags":["test_tag"],"description":"test_file"},"source":"direct-sharing-91"},{"id":34925,"basename":"foo","created":"2018-03-15T14:14:50.074239Z","download_url":"https://open-humans-production.s3.amazonaws.com/member-files/direct-sharing-91/3916dd38-285b-11e8-b199-b6e6950a685d/foo?Signature=cO2Q5l9%2F7CTX8UJ8OYS%2BVYcBCsg%3D&Expires=1521442387&AWSAccessKeyId=AKIAIKNTFUJJTNS6N7HA","metadata":{"tags":["test_tag"],"description":"test_file"},"source":"direct-sharing-91"},{"id":34919,"basename":"foo","created":"2018-03-15T10:42:42.593142Z","download_url":"https://open-humans-production.s3.amazonaws.com/member-files/direct-sharing-91/96e7d93a-283d-11e8-91a1-1aa0cedf2ff2/foo?Signature=DCSxddekw5JqiJIKylXhPCZHsSE%3D&Expires=1521442387&AWSAccessKeyId=AKIAIKNTFUJJTNS6N7HA","metadata":{"tags":["test_tag"],"description":"test_file"},"source":"direct-sharing-91"},{"id":34918,"basename":"foo","created":"2018-03-15T10:41:25.909331Z","download_url":"https://open-humans-production.s3.amazonaws.com/member-files/direct-sharing-91/6936652e-283d-11e8-a7d2-1aa0cedf2ff2/foo?Signature=5cR10LY%2BLDQbMtFMf%2B9DvpA9s8w%3D&Expires=1521442387&AWSAccessKeyId=AKIAIKNTFUJJTNS6N7HA","metadata":{"tags":["test_tag"],"description":"test_file"},"source":"direct-sharing-91"},{"id":34091,"basename":"valid_file.txt","created":"2018-03-12T05:26:45.756016Z","download_url":"https://open-humans-production.s3.amazonaws.com/member-files/direct-sharing-91/f4849aa4-25b5-11e8-b41c-82597e7348b2/valid_file.txt?Signature=g4UtHsX3HNBPZoxHm%2B%2BSLJrIFg4%3D&Expires=1521442387&AWSAccessKeyId=AKIAIKNTFUJJTNS6N7HA","metadata":{"tags":["test_tag"],"description":"test_file"},"source":"direct-sharing-91"}]}]}'}
```

Comment:
@gedankenstuecke are there any decisions on if "real data" should be left in cassettes? I think that we're trying to avoid having this type of real data for the default cassette, but I want to confirm that 😄

Reply:
Yeah, there shouldn't be any real data in the cassettes. Replacing this with place holders would be best.